app/Http/Requests/UpdateEngagementSelectionCriteriaRequest.php

```diff
@@ -72,19 +73,57 @@ public function rules(): array
         ];
     }
 
+    public function attributes(): array
+    {
+        return [
+            'age_brackets' => __('age group'),
+            'age_brackets.*' => __('age group'),
+            'area_types' => __('area type'),
+            'area_types.*' => __('area type'),
+            'disability_types' => __('Disability or Deaf group'),
+            'disability_types.*' => __('Disability or Deaf group'),
+            'ethnoracial_identities' => __('ethnoracial group'),
+            'ethnoracial_identities.*' => __('ethnoracial group'),
+            'first_languages' => __('first language'),
+            'first_languages.*' => __('first language'),
+            'gender_and_sexual_identities' => __('gender or sexual identity group'),
+            'gender_and_sexual_identities.*' => __('gender or sexual identity group'),
+            'ideal_participants' => __('ideal number of participants'),
+            'indigenous_identities' => __('indigenous group'),
+            'indigenous_identities.*' => __('indigenous group'),
```

Comment:
Indigenous should be capitalized, see (for example): https://www2.gov.bc.ca/gov/content/governments/services-for-government/service-experience-digital-delivery/web-content-development-guides/web-style-guide/writing-guide-for-indigenous-content/capitalization-and-formatting-of-indigenous-terms#capitalizing-Indigenous-terms